### ISSUE TITLE
Backtick is escaped wrong

### ISSUE THREAD

**Issue by dimaqq** [ttag-org/ttag#152]
Given this JSX, where the value contains a single literal backtick in the middle

```jsx
<Foo FIXME={t`test\`this`} />
```

`ttag update` produces this block in the translation file:

```po
#: src/Bar.js:13
msgid "test\\`this"
 msgstr ""
```

Where the value contains a backslash and a backtick in the middle.

**Comment by AlexMost** [comment]
You can try fix in `ttag-cli@1.7.11`. Let me know if that works!

**Comment by AlexMost** [comment]
You can try fix in `ttag-cli@1.7.12`. Let me know if that works!

**Comment by dimaqq** [comment]
Yes, it's fixed, thank you! 👍 

Tested with:
```
├─ babel-plugin-ttag@1.7.11
├─ ttag-cli@1.7.13
│  ├─ babel-plugin-ttag@1.7.11
├─ ttag@1.7.13
```
